libraries/config/include/config/network.hpp

```diff
@@ -8,6 +8,12 @@
 
 namespace taraxa {
 
+struct PrometheusConfig {
+  std::string address;
+  uint16_t listen_port = 0;
+  uint16_t polling_interval_ms = 1000;
```

Comment:
btw shouldnt we use const for all config params ?

Reply:
To have const fields here we should change the way we are parsing jsons(use struct constructor for it, not some custom function). Better to be a separate task